.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -88,16 +89,19 @@ body:
     attributes:
       label: Anything in the Host logs that might be useful for us?
       description: >
-        Supervisor Logs can be found under [Configuration panel -> Add-ons, Backup & Supervisor -> System](https://my.home-assistant.io/redirect/supervisor_logs/), then choose Log Provider `Host`.
+        Host Logs can be found in [Settings -> System -> Logs](https://my.home-assistant.io/redirect/logs/)
+        then choose `Host` in the top right.
       render: txt
   - type: textarea
     attributes:
       label: System Health information
       description: >
         **Optional** Copy the full System Health in this text area.
 
-        Can be found in the [Configuration panel -> Settings -> Info](https://my.home-assistant.io/redirect/info/).
-        Use the copy icon on top right and choose `For GitHub`.
+        System Health information can be found in the top right menu in [Settings -> System -> Repairs](https://my.home-assistant.io/redirect/repairs/).
```

Comment:
```suggestion
        System information can be found in [Settings -> System -> Repairs -> System Information (top right menu)](https://my.home-assistant.io/redirect/system_health/),
```

Reply:
Sure, we just have the same link twice then. I was linking to repairs and then providing directions since there was already a button and figured then people would know where it was in the future. But probably overthinking this.